src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Machine> {
```

Comment:
```suggestion
public class BulldozerProducer implements MachineProducer<Bulldozer> {
```
by analogy, the same in other producers